node/tests/ServerModules.test.ts

```diff
@@ -2277,6 +2280,25 @@ describe("Server Module Tests", () => {
                     ).toBeNull();
                 },
             );
+            it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+                "can send transactions_%p",
```

Comment:
I propose to rename, we already have a test with the same name. This specific test could be performed only on a specific server.